app/test_main.py

```diff
@@ -1 +1,62 @@
-# write your code here
+import pytest
+
+from unittest import mock
+
+from app.main import can_access_google_page
+
+
+@pytest.fixture
```

Comment:
you have only one test here so it will be more appropriate to add decorators on top of the function
`@mock.patch("app.main.valid_google_url")`